RxNimble.podspec

```diff
@@ -13,10 +13,25 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = "10.10"
   s.tvos.deployment_target = "9.0"
   s.source       = { :git => "https://github.com/RxSwiftCommunity/RxNimble.git", :tag => s.version }
-  s.source_files  = "Source/**/*.swift"
-  s.dependency "Nimble", "~> 7.0"
-  s.dependency "RxSwift", "~> 4.2"
-  s.dependency "RxBlocking", "~> 4.0"
+  s.default_subspec = "RxBlocking"
```

Comment:
I'm doing this for compability reasons only, but I honestly recommend to use RxTest instead of RxBlocking. We could consider changing the default to `RxTest` in v5.0, perhaps?!